autoscheduler/user_sessions/views.py

```diff
@@ -20,3 +21,14 @@ def set_last_term(request):
         return Response(status=400)
     request.session['term'] = term
     return Response()
+
+@api_view(['GET'])
+def get_full_name(request):
+    """ View that retrieves the first and last name seperated by a space for the user
+    of the current session """
+    user_id = request.session.get('_auth_user_id')
+    if user_id is None:
+        return Response(status=400)
+    user = User.objects.get(pk=user_id)
+    response = {'full_name': user.get_full_name()}
```

Comment:
Really nitpicky but since this is JSON, you should use camelCase instead of snake_case for the key
```suggestion
    response = {'fullName': user.get_full_name()}
```

Reply:
Yeah. You're right. Mocking fetch requests also apparently requires camelCase or it causes an error.